ulib/FStar.BV.fsti

```diff
@@ -197,6 +197,21 @@ val int2bv_div:
     squash (bvdiv #n (int2bv #n x) y == z)
   -> Lemma (int2bv #n (udiv #n x y) == z)
 
+
+(** An unsafe version of 'bvdiv' that does not impose the precondition that the
+      denominator is nonzero. The behaviour of the solver when the denominator
+      is zero is implementation-defined. *)
```

Comment:
I think it would be more accurate to say that bvdiv_unsafe is an uninterpreted function on bv_t n, modeling the corresponding operator from SMT-LIB. When its second argument is zero, the lemma below says that it is equivalent to bvdiv.

Reply:
updated comment.